tests/pickledb_list_tests.rs

```diff
@@ -1,16 +1,48 @@
 #![allow(clippy::float_cmp)]
 
-use pickledb::{PickleDb, PickleDbDumpPolicy, SerializationMethod};
+#[cfg(any(
```

Comment:
I must admit I don't really like adding the supported configuration to every test, it significantly bloats the code and will make adding a new configuration require tons of changes. Unless it's absolutely necessary I'd prefer to avoid it